app/sprinkles/core/src/Util/CheckEnvironment.php

```diff
@@ -351,36 +366,39 @@ public function checkPermissions()
 
     /**
      * Check that PHP meets the minimum required version.
+     *
+     * @return bool True if problem(s) found.
      */
-    public function checkPhp()
+    public function checkPhp(): bool
     {
-        $problemsFound = false;
-
-        // Check PHP version
-        if (version_compare(phpversion(), \UserFrosting\PHP_MIN_VERSION, '<')) {
-            $problemsFound = true;
+        try {
+            $this->checkPhpVersion();
+        } catch (VersionCompareException $e) {
             $this->resultsFailed['phpVersion'] = [
                 'title'   => "<i class='fas fa-code fa-fw'></i> You need to upgrade your PHP installation.",
-                'message' => "I'm sorry, UserFrosting requires version " . \UserFrosting\PHP_MIN_VERSION . ' or greater.  Please upgrade your version of PHP, or contact your web hosting service and ask them to upgrade it for you.',
+                'message' => "I'm sorry, UserFrosting requires version " . $e->getContraint() . ' or greater. Version ' . $e->getVersion() . ' detected. Please upgrade your version of PHP, or contact your web hosting service and ask them to upgrade it for you.',
```

Comment:
Given we are using a semver range, the message should probably be revised. "requires version ^7.2 or greater" doesn't make much sense.